src/Button/ButtonBase.tsx

```diff
@@ -21,6 +23,8 @@ const ButtonBase = forwardRef(
       size = 'medium',
       alignContent = 'center',
       block = false,
+      loading = false,
+      loadingMessage = 'Loading',
```

Comment:
Hi! 

Here's the new place to add props to the documentation: https://github.com/primer/react/blob/main/src/Button/Button.docs.json